src/test/kotlin/blackjack/domain/BlackJackCardMapTest.kt

```diff
@@ -0,0 +1,33 @@
+package blackjack.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+
+class BlackJackCardMapTest {
```

Comment:
도메인 객체와 테스트명을 맞춰주는게 어떨까요! 

```suggestion
class BlackJackCardsMapTest {
```